### ISSUE TITLE
Replicate results

### ISSUE THREAD

**Issue by ayush1997** [felix-laumann/Bayesian_CNN#4]
Hi,
Thanks for the code, I am trying to replicate the results for Bayesian AlexNet (with VI) for CIFAR-10 dataset. I have kept the config as mentioned in the paper i.e. num_epochs = 100, batch_size = 128, optim_type = 'Adam', lr= 0.001. The output is as follows: 

=> Training Epoch #100, LR=0.0002
| Epoch [100/100] Iter[391/391]         Loss: 0.0236 Acc@1: 99.000%00%
| Validation Epoch #100                 Loss: 4.0667 Acc@1: 56.00%
| Elapsed time : 1:34:57

[Phase 4] : Testing model
* Test results : Acc@1 = 61.00%

as opposed to 73% validation accuracy mentioned in paper. Is there anything else I need to change ?

Thanks.

**Comment by ShellingFord221** [comment]
Hi, ayush1997, I wonder how much memory do you need to train the model after 100 epoches?

**Comment by ayush1997** [comment]
@ShellingFord221 ~3-4GB
